docs/projects/even-odd/_posts/2018-12-31-python.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
I'm not sure what our standard is for articles, but I would think we ought to mention every part of the code in the article (including the `main` and the `exit_with_error`) even if we don't go too in depth.

Reply:
Yes! This is true. Always copy the entire solution into the "How to Implement the Solution" section. Check out the template for what I'm talking about: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/CODE_ARTICLE_TEMPLATE.md